src/lido/ids.ts

```diff
@@ -0,0 +1,5 @@
+import { PublicKey } from "@solana/web3.js";
+
+export const LIDO_PROGRAM_ID = new PublicKey("CrX7kMhLC3cSsXJdT7JDgqrRVWGnUpX3gfEfxxU2NVLi");
+export const LIDO_ADDRESS = new PublicKey("49Yi1TKkNyYjPAFdR9LBvoHcUjuPX4Df5T5yv39w2XTn");
+export const ST_SOL_MINT_ADDRESS = new PublicKey("7dHbWXmci3dT8UFYWYZweBLXgycu7Y3iL6trKn1Y7ARj");
```

Comment:
Looks like the account on `mainnet` not upgrade to v2 layout yet, please test with their `testnet` setting as below on local and make sure all test case pass.

```
export const LIDO_PROGRAM_ID = new PublicKey("6RRggYnFe2EcD543QXrE3Wxp1Kgcq8qctwRrNnvjoYsL");
export const LIDO_ADDRESS = new PublicKey("Hcqw2G2FkBhBiDZz3PXFBUKuMQiKMJmXvcfqRaXkyNXF");
export const ST_SOL_MINT_ADDRESS = new PublicKey("Bz2UPiXJmCQYT8XQHaPPXG3fywNp2TENav5dcJdX9q21");
```

Endpoint
```
const connection = new Connection("https://api.testnet.solana.com", {
    commitment: "confirmed",
    confirmTransactionInitialTimeout: 180 * 1000,
  });
```

Reply:
Yep, made a few small changes and it works now :)